lib/bulk_data_test_kit/v2.0.0_client/docs/suite_description.md

```diff
@@ -17,9 +25,9 @@ run them using [this "bulk data client system export" Postman collection](https:
 To run client tests against one of the Postman collections:
 1. Start an Inferno session of the Bulk Data Client test suite.
 2. Navigate to either the export tests group or the delete tests group (depending on which Postman collection you want to use).
-3. Click the "Run Tests" button in the upper right and select the "System Level Export" option for export type.
+3. Click the "Run Tests" button in the upper right, enter any value (e.g. `test`) as the access token, and select the "System Level Export" option for export type.
```

Comment:
One tiny change suggestion. I would maybe change the example token to `SAMPLE_TOKEN` both here and in the Postman collection as default, since that is the access token most Inferno test kits use as the example test token.